Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery.Tests/UnitTests.cs

```diff
@@ -0,0 +1,287 @@
+using NUnit.Framework;
+using System;
+using System.Threading;
+using System.Collections.Generic;
+using Frends.Oracle.ExecuteQuery.Definitions;
+using Newtonsoft.Json.Linq;
+
+namespace Frends.Oracle.ExecuteQuery.Tests
+{
+    /// <summary>
+    /// Docker container for Oracle database is needed to run these tests.
```

Comment:
I think these instructions can be removed, since we have test database available which we can use to run tests. The container + script is only necessary for CI.

Reply:
Fixed